css/css-color/color-mix-percents-01-ref.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 5: color-mix</title>
+<link rel="author" title="Chris Lilley" href="mailto:chris@w3.org">
+<style>
+    .test { background-color: rgb(68.4898% 36.015% 68.3102%); width: 14em; height: 14em; margin-top: 0; margin-bottom: 0;}
```

Comment:
Would be nice to use rounder numbers so that slight floating point inaccuracies don't make the test fail.

Reply:
Maybe, but there are a number of steps to go from the origin colors to the mixing colorspace to the end mixed result, so float inaccuracy can't be entirely eliminated.
Also the spec does say "Values must be rounded towards +∞, not truncated."